lib/src/firestore-to-proto.ts

```diff
@@ -55,7 +55,7 @@ function toMessageInternal<T>(
       console.debug(messageValue);
       continue;
     }
-    if (messageValue) properties[fieldName] = messageValue;
+    if (messageValue !== false) properties[fieldName] = messageValue;
```

Comment:
Please add a test to prevent regresssions.

Reply:
What about boolean `false` values? Shouldn't we instead be checking for `messageValue !== null`?